edx_proctoring/utils.py

```diff
@@ -395,3 +403,58 @@ def categorize_inaccessible_exams_by_date(onboarding_exams, details):
                 non_date_inaccessible_exams.append(onboarding_exam)
 
     return non_date_inaccessible_exams, future_exams, past_due_exams
+
+
+def encrypt_and_encode(data, key):
```

Comment:
The functions added to utils.py are taken from https://github.com/edx/edx-platform/blob/master/lms/djangoapps/verify_student/ssencrypt.py. Other places in edx-platform use similar steps for AES key encryption.